### DISCUSSION TITLE
Handling Auth

### DISCUSSION THREAD

**Discussion by Henry-Do-Su** [frappe/frappe-ui#42] [General]
Hi there,
Whats the best way to deal with Authenticating with Frappe Users on our custom Frontend?
Thanks

**Comment by netchampfaris** [comment]
I use cookie-based session which works flawlessly in Frappe Framework.
For login and signup, there are standard API endpoints.

Here is how I do it in gameplan:
https://github.com/frappe/gameplan/blob/main/frontend/src/data/session.js
https://github.com/frappe/gameplan/blob/main/frontend/src/data/users.js
